modules/brushingandlinking/src/ports/brushingandlinkingports.cpp

```diff
@@ -176,6 +185,13 @@ void BrushingAndLinkingInport::setChanged(bool changed, const Outport* source) {
     Inport::setChanged(changed, source);
 }
 
+void BrushingAndLinkingInport::invalidate(InvalidationLevel invalidationLevel) {
+    if (invalidationLevel == InvalidationLevel::Valid ||
```

Comment:
why this check `invalidationLevel == InvalidationLevel::Valid` ?

Reply:
Assume that it must be possible to set the port to Valid. Otherwise I think that other logic might become broken

Reply:
I believe we use setValid to make it valid